docs/functions.rst

```diff
@@ -1573,6 +1573,41 @@ Example:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
 
+DeleteDataUsingKopiaServer
+----------
+
+This function deletes the snapshot data backed up by the ``BackupDataUsingKopiaServer`` function.
+It creates a new Pod that runs ``delete snapshot`` job created using Kopia Repository Server
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, Yes, `string`, namespace in which to execute the delete job
+   `image`, Yes, `string`, image to be used for running delete job (should contain kopia binary)
+   `backupID`, Yes, `string`, unique snapshot id generated during backup
+   `userPassphrase`, Yes, `string`, user access credentials for kopia repository server
+   `certData`, Yes, `string`, certificate data for kopia repository server
+Example:
+
+Consider a scenario where you wish to delete the data backed up by the
+:ref:`backupdatausingkopiaserver` function.
+For this phase, we will use the ``backupIdentifier`` Artifact provided by backup function.
+
+.. code-block:: yaml
+  :linenos:
+
+  - func: DeleteDataUsingKopiaServer
+    name: DeleteFromObjectStore
+    args:
+      namespace: "{{ .Deployment.Namespace }}"
+      image: ghcr.io/kanisterio/kanister-tools:0.89.0
```

Comment:
Do we really need to have image here? We are always going to use tools image to run the delete command right? Or there are some use cases where this would be helpful?

Reply:
I think if user wants to use some seperate image which have kopia binary, they can put it. But your point seems correct as this is similar like removing `certData` and `userPassphrase` from BP. We could remove it from here and hardcode it in the code

Reply:
Let me know WDYT @viveksinghggits . It seems a Good idea to me

Reply:
Yeah, I want to remove it. But I just looked into the other logical blueprints and for delete actions there we are accepting the image. So let's keep it here for now.

Reply:
Yes, But I was going through some other functions, and there we're using this Image option but we're taking it's value from default kanister images variable. Let's take an example of CopyVolumeData function
```
options := &kube.PodOptions{
		Namespace:    namespace,
		GenerateName: CopyVolumeDataJobPrefix,
		Image:        getKanisterToolsImage(),
		Command:      []string{"sh", "-c", "tail -f /dev/null"},
		Volumes:      map[string]string{pvc: mountPoint},
		PodOverride:  podOverride,
	}
```